src/documents/ChangeApplierAPI.md

```diff
@@ -234,7 +239,8 @@ applier.modelChanged.addListener(pathSpec, listener, namespace)
 applier.modelChanged.removeListener(listener)
 ```
 
-**NOTE**: _This style of listening to changes is **not recommended**. Since such a listener can only be attached once a component and its applier have been fully constructed, it will miss observation of the initial transaction as well as any other model changes that have occurred up to the point where it is registered. This implies that the state of the model that it sees cannot be fully predicted without a knowledge of the entire surrounding of the component tree._
+**NOTE**: _This style of listening to changes is **discouraged**, but may be the right choice in some applications_. For example - the listener to be attached may not be available
+at the time the component is constructed. Note that programmatically attached listeners will miss observation of the initial transaction as any other model changes that have occurred up to the point where they are registered.
```

Comment:
"as any other model changes..." Should this be "as well as any other model changes..." ?